test/dogma/rules/trailing_whitespace_test.exs

```diff
@@ -29,6 +29,26 @@ defmodule Dogma.Rules.TrailingWhitespaceTest do
     ]
   end
 
+  with "windows line endings are okay" do
+    setup context do
+      source = "   'hello'\r\n"
+      <> "'how'\r\n"
+      <> "  'are'\r\n"
+      <> "      'you?'\r\n"
+      <> "'WINDOWS!' \r\n"
+      script = source |> Script.parse( "foo.ex" ) |> TrailingWhitespace.test
```

Comment:
Would we need an example that raise an error? Perhaps this makes it less clear what we are testing here. Perhaps we would want to not use the shouldi matcher for this test, so we can explicitly state why we expect that one error, if we continue to test for it.